src/components/InstanceMonitoring/InstanceMonitoringCard.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import { Icon } from 'carbon-components-react';
+import { InstanceMonitoringContent } from './InstanceMonitoringContent';
+
+export class InstanceMonitoringCard extends Component {
+  render() {
+    const instanceCardProps = {
+      className: 'instance-card',
+    };
+    return (
+      <section {...instanceCardProps}>
+        <InstanceMonitoringContent />
+        <div className="instance-hide">
+          <hr />
+          <span className="instance-hide-link">
+            Hide Summary<Icon
+              className="chevron-icon"
+              name="icon--chevron--up"
```

Comment:
We are deprecating by-name reference for icons. The new approach is importing `iconChevronUp` named export from `carbon-icons` and assign it to `<Icon>` via `icon` prop.